README.md

```diff
@@ -22,20 +21,21 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+To merge a github PR first change directory to a local checkout of the repo,
+and then run (where N is the github PR number):
+
+    $ bundle exec github_merge_sign --pr N
 
 ## Development
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+After checking out the repo, run `bin/setup` to install dependencies. Then, run `bundle exec rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```

Comment:
This change is also in https://github.com/alphagov/paas-github_merge_sign/pull/2 which I'm going to merge first. We'll need to rebase.